crates/sui-indexer-alt/src/metrics.rs

```diff
@@ -287,6 +295,27 @@ impl IndexerMetrics {
                 registry,
             )
             .unwrap(),
+            total_pruner_chunks_attempted: register_int_counter_vec_with_registry!(
+                "indexer_pruner_chunks_attempted",
+                "Number of chunks this pruner attempted to delete",
+                &["pipeline"],
+                registry,
+            )
```

Comment:
noob q but does this report each pipeline's pruner to a separate metric, or into a single metric?

Reply:
Each one in a separate metric (labeled by the pipeline name).